Earthfile

```diff
@@ -214,6 +209,11 @@ mocks:
         SAVE ARTIFACT $mockfile AS LOCAL $mockfile
     END
 
+# cli-unit-test runs the unit tests for the CLI commands in ./cmd/earthly
+cli-unit-test:
```

Comment:
These unit tests are also triggered from the +unit-test target but it's nice to be able to call them separately for speed

Reply:
I don't think this is needed since it should be possible to use `+unit-test --pkgname=./cmd/earthly/...` instead.

Reply:
Ahh, that is convenient :D I'll remove that